README.md

```diff
@@ -1,6 +1,6 @@
 # getpasswd in Go [![GoDoc](https://godoc.org/github.com/howeyc/gopass?status.svg)](https://godoc.org/github.com/howeyc/gopass) [![Build Status](https://secure.travis-ci.org/howeyc/gopass.png?branch=master)](http://travis-ci.org/howeyc/gopass)
 
-Retrieve password from user terminal input without echo
+Retrieve password from user terminal or piped input without echo.
```

Comment:
With ssh/terminal existing, simply getting the password isn't a differentiator.  I added this but it could/should also be stated that you can use a silent or masked format; those (masking and piping) are the things more unique to this lib.